musicbot/commands/music.py

```diff
@@ -48,10 +48,12 @@ async def _play_song(self, ctx, *, track: str):
 
         if song.origin == linkutils.Origins.Default:
 
-            if audiocontroller.current_song != None and len(audiocontroller.playlist.playque) == 0:
-                await ctx.send(embed=song.info.format_output(config.SONGINFO_NOW_PLAYING))
-            else:
+            sett = utils.guild_to_settings[self.guild]
```

Comment:
Songs will not play because the music class has no self.guild

```suggestion
            sett = utils.guild_to_settings[ctx.guild]
```